invenio_opendata/testsuite/data/cms/cms-vm.xml

```diff
@@ -0,0 +1,97 @@
+      <subfield code="y">CMS Virtual Machines - Known Issues and Limitations</subfield>
+    </datafield>
+    <datafield tag="567" ind1=" " ind2=" ">
+      <subfield code="a">The contextualisation scripts used for setting up the CMS VM images are available at</subfield>
+      <subfield code="u">LINK</subfield>
+    </datafield>
+    <datafield tag="581" ind1=" " ind2=" ">
+      <subfield code="a">Please follow the tutorial on how to use the CMS Virtual Machine</subfield>
+      <subfield code="u">http://opendata.cern.ch/VM/CMS#2011</subfield>
+      <subfield code="y">CMS Virtual Machines: How to install</subfield>
+    </datafield>
+    <datafield tag="583" ind1=" " ind2=" ">
+      <subfield code="a">A validation report of the CMS VM can be found at</subfield>
+      <subfield code="u">http://opendata.cern.ch/VM/CMS/validation/report#2011</subfield>
+      <subfield code="y">Testing the CernVM</subfield>
+    </datafield>
+    <datafield tag="693" ind1=" " ind2=" ">
+      <subfield code="a">CERN-LHC</subfield>
+      <subfield code="e">CMS</subfield>
+    </datafield>
+    <datafield tag="964" ind1=" " ind2="0">
+      <subfield code="c">Run2011A</subfield>
+    </datafield>
+    <datafield tag="980" ind1=" " ind2=" ">
+      <subfield code="a">CMS-Tools</subfield>
+    </datafield>
+   </record>
+  <record>
+    <controlfield tag="001">251</controlfield>
+    <datafield tag="100" ind1=" " ind2=" ">
+      <subfield code="a">TEXT</subfield>
+    </datafield>
+     <datafield tag="245" ind1=" " ind2=" ">
+      <subfield code="a">Contextualisation scripts to create the CMS VM Image, for 2011 data release</subfield>
+     </datafield>
+    <datafield tag="260" ind1=" " ind2=" ">
+     <subfield code="b">CERN Open Data Portal</subfield>
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="264" ind1=" " ind2="0">
+      <subfield code="c">2011</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">The scripts provided create the CMS virtual machine image based on CernVM (cernvm.cern.ch), with Scientific Linux CERN 6 (slc6).</subfield>
+    </datafield>
+    <datafield tag="538" ind1=" " ind2=" ">
+      <subfield code="a">Use this with a Micro-CernVM available at</subfield>
+      <subfield code="u">http://cernvm.cern.ch/portal/downloads</subfield>
+    </datafield>
+    <datafield tag="581" ind1=" " ind2=" ">
+      <subfield code="a"><p> First, you need to run a normal Micro-CernVM, using your own context so you can login. Copy the attached files to the running VM and then 
+      login.<p>Inside the VM, you first need a copy of the latest Micro-CernVM release as a template for the Open Data image. Download it, inside the running VM, from: 
+      http://cernvm.cern.ch/releases/ucernvm-images.1.18-13.cernvm.x86_64/ucernvm-slc5.1.18-13.cernvm.x86_64.hdd <br/>placing it in the same directory as the other files.<br/>Run 
```

Comment:
The base image currently in use is at:

[http://cernvm.cern.ch/releases/testing/ucernvm-prod.2.5-2.cernvm.x86_64.hdd](url)